apollo-livedata-support/src/main/java/com/apollographql/apollo/livedata/LiveDataApollo.kt

```diff
@@ -0,0 +1,227 @@
+package com.apollographql.apollo.livedata
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.Observer
+import android.support.annotation.NonNull
+import com.apollographql.apollo.ApolloCall
+import com.apollographql.apollo.ApolloPrefetch
+import com.apollographql.apollo.ApolloQueryWatcher
+import com.apollographql.apollo.ApolloSubscriptionCall
+import com.apollographql.apollo.api.Response
+import com.apollographql.apollo.api.internal.Utils.checkNotNull
+import com.apollographql.apollo.cache.normalized.ApolloStoreOperation
+import com.apollographql.apollo.exception.ApolloException
+import java.util.concurrent.atomic.AtomicBoolean
+
+/**
+ * The LiveDataApollo class provides methods for converting ApolloCall, ApolloPrefetch and ApolloWatcher types to LiveData
+ * sources.
+ */
+@Suppress("unused")
+object LiveDataApollo {
```

Comment:
In Kotlin, The default visibility, used if there is no explicit modifier, is public.

Reply:
In Kotlin, The default visibility, used if there is no explicit modifier, is public.
Do you mean the `object` should be `internal`?

Reply:
Yeah make `internal object LiveDataApollo`. I don't see the reason why to expose it if it used internally only

Reply:
:white_check_mark: Ah, I think it is better too. Because In Java also can use` ApolloExtension` like this. 
```java
ApolloExtensionKt.toLiveData(call)
```
It will be fixed!

Reply:
:white_check_mark: Ah, I think it is better too. Because In Java also can use` ApolloExtension` like this. 
```java
LiveDataApolloExtensionKt.toLiveData(call);
```
So I will fix the extension file package : `ApolloExtension.kt` to `LiveDataApolloExtension.kt`.
And object LiveDataApollo will be internal.